extensions/azurecore/tsconfig.json

```diff
@@ -1,14 +1,22 @@
 {
-	"extends": "../shared.tsconfig.json",
+	"compileOnSave": true,
```

Comment:
why are you reverting this?

Reply:
Is this required from requiring vscode directly? would just removing the .d.ts reference in the code fix compiling without needing this reverted?

Reply:
Importantly, the shared tsconfig contains a bunch of strict compile options that help ensure strict (good practice) coding guidelines, so by revert this, we are no longer enforcing those.

Reply:
Thanks - I had been trying to fix the failing build and tried changing (again, to another of our extensions) .tsconfig settings that I knew a) used VSCode module and b) did compile. Having reverted the file, it looks like it still works. So it's a problem with newer VSCode module versions where they always bundle in a vscode.d.ts file, causing duplication problems. 

Anyhow, this should be reverted in the next iteration